src/discordtipbot/discordbot.cr

```diff
@@ -3,6 +3,8 @@ require "./utilities"
 class DiscordBot
   include Utilities
 
+  setter config : Config
```

Comment:
Does this setter actually get used anywhere now...?

Reply:
Nope. Nice catch